flexible_event_config.md

```diff
@@ -47,7 +50,165 @@ In general, the approach here is to more flexibly encode the output of the API f
+  "trigger_data": [0],
+  "event_report_windows": {
+    // 1 day, 2 days, 3 days, 5 days, 7 days, 10 days represented in seconds
+    "end_times": [86400, 172800, 259200, 432000, 604800, 864000]
+  },
+}
+```
+
+## Attribution Rate Limit
+
+Attribution rate-limit behavior will be based on the number of reports generated
+rather than the number of triggers that lead to report generation. For example:
+The first report in the
+["Reporting trigger value buckets"](#reporting-trigger-value-buckets) example
+will be counted as 1 contribution towards the
+[max attribution per rate-limit window](https://wicg.github.io/attribution-reporting-api/#max-attributions-per-rate-limit-window)
+limit despite it being the result of contributions from 3 triggers.
+
+## Privacy considerations
+
+We will publish an [algorithm](https://github.com/WICG/attribution-reporting-api/tree/main/ts#flexible-event) which computes the number of output states for a given source registration. From this we will be able to:
+
+* Compute a randomized response algorithm across the entire output space
+* Set the noise level to satisfy a certain epsilon level via a randomized response mechanism
+* Verify that the privacy parameters (like information gain) are within a given threshold, and fail registration if they are not
+
+With these pieces we can ensure that these extensions do not exceed certain privacy parameters. Additionally, it allows callers to fine-tune the noise added to the API e.g. by specifying different kinds of output domains. For example, navigation sources that only need 1 bit of trigger data and 1 reporting window can use the same noise level as event sources.
+
+Beyond setting noise levels, we will have some parameter limits to avoid large computation costs and avoid configurations with too many output states (where noise will increase considerably). Here is an example set of restrictions (feedback always welcome):
+
+* Maximum of 20 total reports, globally and per `trigger_data`
+* Maximum of 5 possible reporting windows per `trigger_data`
+* Maximum of 32 trigger data cardinality (not applicable for Phase 1: Lite Flexible Event-Level)
+
+Be mindful that using extreme values here may result in a large amount of noise, or failure to register if privacy levels ([information gain](https://github.com/WICG/attribution-reporting-api/blob/main/params/chromium-params.md)) are exceeded. The [flexible-event script](https://github.com/WICG/attribution-reporting-api/tree/main/ts#flexible-event) can be used to analyze different configurations that fall within the privacy levels.
+
+
+## Ideas for future iteration
+
+### Summary Buckets and Multiple Trigger Specs
+
+The trigger registration could also support three additional fields: `trigger_specs`, `summary_operator` and `summary_bucket` that would allow each source regisration supporting multiple trigger specs per registration and the ability to summarize report values and trade off noise with data granularity.
```

Comment:
```suggestion
The trigger registration could also support three additional fields: `trigger_specs`, `summary_operator`, and `summary_bucket` that would allow each source regisration supporting multiple trigger specs per registration and the ability to summarize report values and trade off noise with data granularity.
```